docs/source/legacy-dev-docs.rst

```diff
@@ -213,103 +211,6 @@ To acquire the configured credentials the following can be used:
     }
     ...
 
-PropertyProvider
```

Comment:
This shouldn't be removed, as above you'll just need to pull the latest changes from master in.

Reply:
This is I believe the one example of where changed/edits were made on the master. I rebased. However when I try including the content in PropertyProvider, I get an warning/error on the build.  Unsure how to proceed on this one.